### ISSUE TITLE
nested `CategoryList` does not renders nested list properly.

### ISSUE THREAD

**Issue by AxDSan** [lovata/oc-shopaholic-plugin#23]
This is my backend `Trades` Page
![](https://my.mixtape.moe/zavkiz.png)

This is my rendered view:
![](https://my.mixtape.moe/vmaqhw.png)

This is my Category List (backend):
![](https://my.mixtape.moe/kzbudt.png)

I don't know what I'm doing wrong here, I also have removed the portion of the code which checks if there are any products in the respective categories 👍

**Comment by kharanenka** [comment]
I can not reproduce this bug.
Please, clear cache and debug plugins/lovata/shopaholic/classes/item/CategoryItem.php class, line 140. 
```php
$this->obElement->children()
            ->active()
            ->orderBy('nest_left', 'asc')
            ->lists('id');
```
We stores in the cache the list of IDs of child categories in this row.

**Comment by AxDSan** [comment]
I'm Sorry for hitting the `Panic Button` xD apparently it was something to do with the cache! thanks!
![](https://i.imgur.com/SqLv6Y1.png)

**Comment by kharanenka** [comment]
Thanks for the help! This is not good if you need to manually clean the cache. Bug fixed in version 1.2.2

**Comment by AxDSan** [comment]
@kharanenka My pleasure to help you out !
